cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/sv/GMLSvDataReader.java

```diff
@@ -147,7 +149,9 @@ private void processGmlSvFile() {
                 String patientId = cvrSampleListUtil.getSamplePatientId(to_add.getSample_ID());
                 // check that matching patient id can be found from patient-sample mapping
                 // and whether patient is in new dmp germline patients (to prevent duplicates)
-                if (!Strings.isNullOrEmpty(patientId) && !cvrSampleListUtil.getNewDmpGmlPatients().contains(patientId)) {
+                if (!Strings.isNullOrEmpty(patientId)
+		        && !cvrSampleListUtil.getNewDmpGmlPatients().contains(patientId)
+			&& cvrSampleListUtil.getPortalSamples().contains(to_add.getSample_ID())) {
```

Comment:
Added check - don't process SV record if sample it belongs to is not in portalSamples.